iree/compiler/Dialect/LinalgExt/Transforms/test/convert_to_loops.mlir

```diff
@@ -94,3 +94,175 @@ func @sort_multi(%arg0: memref<128xf32>, %arg1: memref<128xi32>) {
 // CHECK:               memref.store %[[V4]], %[[BUF2]][%[[ARG2]]]
 // CHECK:               memref.store %[[V3]], %[[BUF2]][%[[T2]]]
 // CHECK:             }
+
+// -----
+
+func @scatter_update_scalar_1D(
```

Comment:
Nit: Could we make all these tests work with dynamic shapes. Its a bit more of Lit testing goo, but it keeps us more honest in terms of dynamic shape codegen.

Reply:
Sure, let me add more tests for dynamic shapes.

Reply:
Added three more test cases, which covers `index_depth=1/scalar_update` case, `index_depth=2/scalar_update` case, and `index_depth=1/slice_update` case.